src/common/defs.rs

```diff
@@ -11,6 +11,11 @@ pub(crate) const BLKID_CMD: &str = "blkid";
 pub(crate) const EFIBOOTMGR_CMD: &str = "efibootmgr";
 pub(crate) const DD_CMD: &str = "dd";
 
+/* The mdtd_debug tool is used on Xavier NX devices to clear and write the QSPI
+   with the blob included in the target OS image
+*/
+pub(crate) const MTD_DEBUG_CMD: &str = "mtd_debug";
```

Comment:
This is another of my more generic suggestions. One problem I think takeover suffers from already is assuming that the future maintainers know everything the original developer knew.  I often find hard to understand what exactly some part of the code is doing, and especially why.

And here's one case in which I think we are making this situation worse. I personally don't know what `mtd_debug` is, and a quick search gives me only a vague idea of what it is. So, I think a few lines of comments here could be very helpful in the long run. A link to wherever this tool is documented could also help a lot.

Reply:
Sure, I added more details on mtd_debug and the mtd-utils package in https://github.com/balena-os/takeover/pull/66/commits/038b56c8215747041ec31ee7a6fce45b852f6f97

Reply:
Anything else I should add here?

Reply:
Nope! That was a great improvement, thanks!